.github/pull-request-template.md

```
@@ -1,25 +1,25 @@
-### Pre-flight Checklist
-
-<!-- Put an 'x' in all boxes that apply -->
-
-- [ ] Changes are limited to a single feature, bugfix or chore (split larger changes into separate PRs)
-- [ ] `bun check` without any issues
-- [ ] I have reviewed [contributor guidelines](https://github.com/techwithanirudh/discord-ai-bot/blob/main/CONTRIBUTING.md)
-
-### Additional Notes
-
-<!-- Add any additional notes for reviewers -->
+### Description
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)